Deprecate installing harness tools in php moving upstream

andytson-inviqa opened the pull request:
and avoid repeating installation whilst deprecated

andytson-inviqa committed:
Deprecate installing harness tools in php moving upstream

and avoid repeating installation whilst deprecated

andytson-inviqa commented:
the redis-cli seems to be unreliable as well, so I'll have a think about that (perhaps, if we get it working, we put the latest redis-cli release in the upstream image if it's backwards compatible with older servers)

kierenevans commented on src/_base/docker/image/php-fpm/Dockerfile.twig:
```diff
@@ -9,13 +9,15 @@ FROM {{ @('docker.image.php-fpm') }}
 {%- set install_extensions=@('php.install_extensions')|merge(@('php.fpm.install_extensions'))|filter(v => v is not empty) %}
-{%- if install_extensions %} \
- && cd /root/installer \
+{%- if install_extensions %}
```
```suggestion
{%- if install_extensions %} \
```
Should fix the build error - this gets put on the same line as `chmod +x /usr/local/bin/mhsendmail` by twig.

Perhaps there's a better way to do it?

andytson-inviqa replied:
The better way is to not use - in twig as it already does trim a bit anyway. I've removed most of them used there, and added one. makes xdebug the line after extensions rather than having 2 lines gap if <3, and still has an empty line above it if no extensions

kierenevans commented:
Is redis-cli an issue due to using an alpine redis image with a debian-based php image?
Latest redis sounds good though.

andytson-inviqa committed:
Remove redis-cli copying from alpine image

No-one's going to switch to redis debian buster version just for this, so it's better to get the one that works from the upstream version (and upgrade that)

andytson-inviqa committed:
Fix whitespace removal
